easybuild/easyconfigs/n/NewHybrids/NewHybrids-1.1_Beta3-GCC-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+# Author: Denis Krišťák (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'NewHybrids'
+version = '1.1_Beta3'
+
+homepage = 'http://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs'
+description = """This implements a Gibbs sampler to estimate the posterior probability that genetically sampled
+individuals fall into each of a set of user-defined hybrid categories."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/eriqande',
+        'repo_name': 'newhybrids',
+        'commit': '6fc8fd9ca6e64ee0be08af12ec8bfe73a7de00cc',
+        'recursive': True,
+    }
+}]
+
+skipsteps = ['configure']
+
+build_cmd = './Compile-with-no-gui-Linux.sh'
```

Comment:
How does this script compile stuff? Which compiler command, and in particular, which compile *options* does it use? Does it honor `$CFLAGS` & co?

Reply:
i dont think so, content of the file:
```gcc -O3 -o newhybrids-no-gui-linux.exe \
  src/GLUT_for_NewHybrids.c  \
	src/DataInput.c  \
	src/DataOutput.c  \
	src/NewHybrids.c  \
	src/RunWOG_func.c \
	eca-shared/src/ECA_Opt3.c \
	eca-shared/src/MathStatRand.c \
	eca-shared/src/MCTypesEtc.c \
	eca-shared/src/com.c \
	eca-shared/src/linpack.c \
	eca-shared/src/ranlib.c \
	eca-shared/src/ECA_ReadData.c \
	eca-shared/src/ECA_MemAlloc.c \
	eca-shared/src/ECA_utilities.c \
	-I eca-shared/include \
	-I src \
	-DCOMPILE_NEW_HYB_WITH_NO_GUI \
        -lm
```

Reply:
installation takes 4 seconds on gligar, so no need i guess